go/vt/vtgate/engine/plan.go

```diff
@@ -76,6 +77,15 @@ func (p *Plan) MarshalJSON() ([]byte, error) {
 		instructions = &description
 	}
 
+	tablesUsed := make([]string, len(p.TablesUsed))
+	for i, table := range p.TablesUsed {
+		if table.Qualifier.NotEmpty() {
+			tablesUsed[i] = fmt.Sprintf("%s.%s", table.Qualifier.String(), table.Name.String())
+		} else {
+			tablesUsed[i] = table.Name.String()
+		}
```

Comment:
```suggestion
		tablesUsed[i] = sqlparser.String(table)
```

Reply:
@systay the problem with that is it adds backticks around keyspace names and tables. Which I guess isn't really a problem if we're OK with updating all the `planbuilder` tests and any other tests that use this info?